src/shared/appUtils.ts

```diff
@@ -0,0 +1,63 @@
+import BaseUtils from "./baseUtils";
```

Comment:
As these files are mainly to support profile fetching and not necessarily shared between any other commands, Lets create a folder to support profile

Reply:
Yes, those files mainly support profiles fetching now. But will be necessary when implementing the same feature for permission sets.

Reply:
okie,lets move it to another folder still, shared is becoming too cluttered

Reply:
Moved to shared profile utils